dependencies/maven/artifacts.bzl

```diff
@@ -39,8 +39,7 @@ artifacts = [
     "io.netty:netty-all",
     "junit:junit",
     "org.rocksdb:rocksdbjni",
-    "org.rocksdb:rocksdbjni-dev",
-    "org.rocksdb:rocksdbjni-dev-linux",
```

Comment:
the `-linux` dev distribution is broken, so we don't publish the maven jar for it anymore and cannot depend on it.